components/layermanager/layermanager-metadata.service.js

```diff
@@ -143,7 +170,23 @@ export default function (
               '0': caps.Service,
             });
           }
-
+          //Identify max resolution of layer. If layer has sublayers the heighest value is selected
+          $timeout(() => {
+            if (layer.get('MaxScaleDenominator')) {
+              layer.set(
+                'maxResolution',
+                me.roundToHundrets(layer.get('MaxScaleDenominator'))
+              );
+              return;
+            }
+            const maxScale = me.searchForScaleDenominator(
+              layer.getProperties()
```

Comment:
Why not just layer without getProperties?

Reply:
Not sure why I went this way. ty

Reply:
searchForScaleDenominator is recursive while sublayers are not instances of ol.layer so they dont have getProperties method